assets/src/edit-story/components/canvas/useCanvasSelectionCopyPaste.js

```diff
@@ -82,10 +82,8 @@ function useCanvasSelectionCopyPaste(container) {
       const htmlContent = selectedElements
         .map(({ type, ...rest }) => {
           // eslint-disable-next-line @wordpress/no-unused-vars-before-return
-          const { Output } = getDefinitionForType(type);
-          return renderToString(
-            <Output element={rest} box={{ width: 100, height: 100 }} />
-          );
+          const { Save } = getDefinitionForType(type);
+          return renderToString(<Save {...rest} />);
```

Comment:
Accidental change here? Should still use `Output` here with `element` and `box` props

Reply:
I think this is a merge issue. I will fix.

Reply:
Fix https://github.com/google/web-stories-wp/pull/210/commits/a48e6940e2e90a3d2a3a40cfc3baef6aba56dde9